libs/core/src/components/sage-chip/sage-chip.tsx

```diff
@@ -0,0 +1,89 @@
+import { Component, Host, h, Prop, Event, EventEmitter } from '@stencil/core';
+
+@Component({
+  tag: 'sage-chip',
+  styleUrl: 'sage-chip.scss',
+  shadow: true,
+})
+export class SageChip {
+  /**
+   * Sets the color scheme of the chip.
+   */
+  @Prop() sentiment: 'accent' | 'danger' | 'info' | 'neutral' | 'success' | 'warning' = 'neutral';
+
+  /**
+   * Determines whether a dot should be displayed on the chip.
+   */
+  @Prop() dot: boolean;
```

Comment:
For the boolean props, is it possible to set their defaultValue as `false` if it doesn't currently have one?